sympy/codegen/tests/test_ast.py

```diff
@@ -251,14 +251,14 @@ def test_For():
 
 def test_none():
     assert none.is_Atom
-    assert none == none
```

Comment:
This is testing the `__eq__` machinery, so this should not be changed. Note that this is `none`, which is a sympy object and not the python builtin `None`.